docs/reference/catalog-reference.md

```diff
@@ -1,6 +1,360 @@
 # Catalog
 
-::: up42.catalog.Catalog
-    selection:
-        inherited_members: True
-        members: {{ funcs_catalog }}
+The Catalog class enables access to the UP42 [catalog functionality](../../catalog/).
+
+```python
+catalog = up42.initialize_catalog()
+```
+
+This class also inherits functions from the [CatalogBase](catalogbase-reference.md) class.
+
+## Scenes
+
+### construct_search_parameters()
+
+The `construct_search_parameters()` function allows you to fill out search parameters when searching for catalog imagery.
+
+```python
+construct_search_parameters(
+    geometry,
+    collections,
+    start_date,
+    end_date,
+    usage_type,
+    limit,
+    max_cloud_cover,
+)
+```
+
+The returned format is `dict`.
```

Comment:
@dustgalactic I thought we agreed on `The returned data type is ...`

Reply:
Not for now, it'll be done in a different iteration. @seedlit